### PR TITLE
Tests for no select permissions

### PR DESCRIPTION
**Pull request by xBis7**

### CODE REVIEW

**Comment by xBis7** [comment]
This PR is a draft because it's waiting on the Hive and Ranger commit SHAs. The tests are not passing yet but that's a Ranger/Hive issue. Apart from the update on the commit SHAs, there are no more changes incoming for this patch.

File: trino-hms-hdfs-ranger/testlib.sh [hunk 1]
```diff
@@ -40,10 +40,10 @@ configureHiveVersion() {
     echo "Configuring project for Hive 3."
     echo ""
     # Hive branch: 'branch-3.1-build-fixed'
-    HIVE_COMMIT_SHA="2f7682e412caf81202e223bd2fe9c50b5c1188f4"
+    HIVE_COMMIT_SHA="2f7682e412caf81202e223bd2fe9c50b5c1188f4"  # TODO: update.
```
Comment (mladjan-gadzic) [inline trino-hms-hdfs-ranger/testlib.sh:43]: Please don't forget to update this once you're done.

**Review by mladjan-gadzic — APPROVED** [review]
@xBis7 thanks for the work! I've left inline comment as a remainder. I'll approve this PR so you can continue with merge when you're done with changes.

**Comment by xBis7** [comment]
I've run `run_all.sh`, `test_database.sh` and `test_schema.sh` and all tests are passing.

**Pull request merged**